manifests/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -42,12 +42,14 @@ spec:
 {{- if .Values.global.proxy.enableCoreDump }}
       initContainers:
         - name: enable-core-dump
-{{- if contains "/" .Values.global.proxy_init.image }}
-          image: "{{ .Values.global.proxy_init.image }}"
+{{- if contains "/" .Values.global.proxy.image }}
```

Comment:
I'm not entirely sure this should change. I don't know of any use cases for having proxy-init != proxyv2 but there might be?

Reply:
I changed this to match ingress gateway deployment.  Proxy_init is deprecated, and we are just using the proxy image for init container.